### PR TITLE
Fixes for sorting bugs #25, and #26

### LINKED ISSUE #25: Change default column sorting to DKP
I think that more users prefer to sort by DKP than Class.
The change would be made in Core.lua.
Current:
```lua
core.currentSort = "class"
```
Proposed:
```lua
core.currentSort = "dkp"
```

### LINKED ISSUE #26: Sorting method not always preserved when changing active team
Changing the current team sometimes (but not always) sorts the new team entries alphabetically, disregarding current sorting method. The sorting method is remembered (core.currentSort is not changed), which causes the order to reverse if the user selects the same sort type.

Reproduction steps:
1. Reload UI
2. Sort by DKP
3. Change active team
4. Change to previously active team
(Note that sorting is now alphabetical)
5. Sort by DKP
(Note that sorting is now by reverse DKP)

I was able to fix this with a change in CommunityDKP.lua, after line 493:
```lua
CommDKP:SortDKPTable(core.currentSort);
CommDKP:SortDKPTable(core.currentSort);
```
I don't feel a performance difference with the extra sorting.

### PR DESCRIPTION
**Pull request by Anonomit**
Default sorting changed to dkp, and sorts table after dropdown changes the working table

### CODE REVIEW

**Comment by Anonomit** [comment]
#25 is actually an enhancement, not a bug.

File: Modules/Bidding.lua [hunk 1]
```diff
@@ -747,34 +747,37 @@ function CommDKP_Register_ShiftClickLootWindowHook()      -- hook function into
...
-                  ReloadUI();
-                end,
-                timeout = 0,
-                whileDead = true,
-                hideOnEscape = true,
-                preferredIndex = 3,
-              }
-              StaticPopup_Show ("SUGGEST_RELOAD")
-            end
+        local lootSlot = getglobal("ElvLootSlot"..i)
```
Comment (Vapok) [inline Modules/Bidding.lua:750]: Is this just a formatting change?  Trying to determine if there's an actual change here or not.

Comment (Anonomit) [reply Modules/Bidding.lua:750]: It probably should have been a different pull request, but that's just adding a check to make sure getglobal actually returns a value before it tries to use it. The indenting was bizarre there, which I also fixed. It went from:
`getglobal():dothings()`
to
`local thing = getglobal()
if thing then
  thing:dothings()
end`
I think it's probably caused by an addon that's speeding up autoloot, but I would get a harmless error right there sometimes.

Comment (Vapok) [reply Modules/Bidding.lua:750]: Fantastic, I love little fixes like this. Thank you!

**Review by Vapok — APPROVED** [review]

**Pull request merged**